test/CreatorToken.t.sol

```diff
@@ -302,6 +314,7 @@ contract Selling is CreatorTokenTest {
   function test_RevertIf_SellerIsNotTokenOwner(address _owner, address _seller) public {
     _assumeSafeBuyer(_seller);
     buyAToken(_owner);
+    vm.assume(_owner != _seller);
```

Comment:
We should do the assume before the `buyAToken` portion of the test to avoid wasting time on a test execution that will ultimately be thrown out.

Reply:
fixed in e28ac5b30af2bb086482debf189123a96e11d220